docs/get_started/introduction.rst

```diff
@@ -115,8 +113,9 @@ You can save the code below into a Python file and run it.
   MLC LLM Python API
 
 This code example first creates an :class:`mlc_llm.Engine` instance with the the 4-bit quantized Llama-2 model.
-**The Python API of** :class:`mlc_llm.Engine` **if fully compatible with OpenAI API**,
-which means you can use :class:`mlc_llm.Engine` in the same way of using `OpenAI's Python package <https://github.com/openai/openai-python?tab=readme-ov-file#usage>`_
+**We design the Python API ** :class:`mlc_llm.Engine` **to align with OpenAI API**,
```

Comment:
Need to remove the whitespace here or otherwise it doesn't properly display.
```suggestion
**We design the Python API** :class:`mlc_llm.Engine` **to align with OpenAI API**,
```